### ISSUE TITLE
Make it possible to intercept protoc-jar debug output

### ISSUE THREAD

**Issue by bjaglin** [os72/protoc-jar#74]
Even though there is a dedicated overload for capturing `protoc` stdout/stderr since https://github.com/os72/protoc-jar/pull/49, `protoc-jar`'s own debug logs are still being [dumped to `System.out`](https://github.com/os72/protoc-jar/blob/de935a1/src/main/java/com/github/os72/protocjar/Protoc.java#L431-L433).

See https://github.com/thesamet/sbt-protoc/pull/121#discussion_r344465994 for where it would be beneficial.

**Comment by os72** [comment]
Yes good point. But it may require a larger refactoring, the options are getting a bit out of hand..
